src/DIRAC/WorkloadManagementSystem/JobWrapper/JobWrapper.py

```diff
@@ -119,6 +119,7 @@ def __init__(self, jobID=None, jobReport=None):
         self.defaultErrorFile = gConfig.getValue(self.section + "/DefaultErrorFile", "std.err")
         self.diskSE = gConfig.getValue(self.section + "/DiskSE", ["-disk", "-DST", "-USER"])
         self.tapeSE = gConfig.getValue(self.section + "/TapeSE", ["-tape", "-RDST", "-RAW"])
+        self.failoverRequestDelay = gConfig.getValue(self.section + "/FailoverRequestDelay", 45) * 60
```

Comment:
Watch out, for once the doc is correct ! 
https://github.com/DIRACGrid/DIRAC/blob/591db8ccc25bfbfd34e2c18fd4ab4f6160cdce0e/src/DIRAC/RequestManagementSystem/Client/Request.py#L334

This is minutes already

Reply:
OK, I'll fix